vendor/github.com/ceph/go-ceph/rados/conn.go

```diff
@@ -5,9 +5,11 @@ package rados
 // #include <rados/librados.h>
 import "C"
 
-import "unsafe"
-import "bytes"
-import "fmt"
+import (
```

Comment:
We should also push this into our go-ceph and add a test to make sure this works as expected.

Changing the vendored stuff directly feels a little odd. We can push this for now, but we should see if we can fork ceph/go-ceph to digitalocean/go-ceph and make these changes there (and then vendor that repo in).

Reply:
Agree that the approach should be to send PR to the upstream, and I have a plan for that. But beforehand, I need to figure out how to setup a local environment in order to successfully run the tests. This change should be simple but going through the process might take some time.